cappy.py

```diff
@@ -42,20 +44,21 @@ def make_promise():
 
 
 class JsonDataStream(object):
-    """Consumes bytes and produces JSON objects"""
+    """Consumes bytes and produces JSON objects."""
 
     def __init__(self):
         self.depth = 0
         self.buf = ''
 
     def receive(self, data):
-        """Process incoming bytes, returning a json object if 
+        """
+        Process incoming bytes, returning a list of dicts.
+
+        Each dictionary represents a JSON object from the wire. If no complete
+        JSON objects are contained in the bytes from the wire the list will be
+        empty.
```

Comment:
Could you add the `Args` and `Returns` sections back? As far as I can tell Pep257 doesn't say anything about that, and having them is really useful.